### PR TITLE
Controller acceleration

### PR DESCRIPTION
**Pull request by pelikhan**
Basic blocks to support an accelerometer in arcade. Note that the "onGesture" event runs the handler in the "update" frame handler.
image

### CODE REVIEW

File: libs/controller/accelerometer.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+     * x=0, y=0 and z=-1023)
+     * @param dimension the axis along which the acceleration if measured
+     */
+    //% blockId=ctrlaccelerationvalue block="acceleration (mg)|%NAME"
+    //% parts="accelerometer"
+    //% dimension.fieldEditor="gridpicker"
+    //% dimension.fieldOptions.width=180
+    //% dimension.fieldOptions.columns=2
+    //% group="Extras"
+    export function acceleration(dimension: ControllerDimension): number {
```
Comment (mmoskal) [inline libs/controller/accelerometer.ts]: Can't we have controller.accelerationX, controller.accelerationY, ... and collapse them into one block like we do for sprite properties?

**Comment by darzu** [comment]
What does "vibrate" do? How is it different than camera shake?

**Comment by pelikhan** [comment]
Camera shake is a visual effect, vibrate turns on a vibrating motor.

File: libs/controller/accelerometer.ts (hunk 1 above)
Comment (pelikhan) [reply libs/controller/accelerometer.ts]: Only support on class getters/fields.

**Pull request merged**